reveal/run.js

```diff
@@ -88,8 +91,23 @@
     });
   }
 
-  function hasRemoteQueryParameter() {
-    return window.location.search.match(/[?&]remote[&]?/i);
+  function selectWidth(queryParams) {
+    if (queryParameterIsPresent(window.location.search, 'wide')) return 1413; // 16:9
+    else if (queryParameterIsPresent(window.location.search, 'wide10')) return 1272; // 16:10
+    else return 1124; // ~4:3
+  }
+
+  function cycleWideModes() {
+    var queryString = window.location.search;
+    if (queryParameterIsPresent(queryString, 'wide')) {
```

Comment:
I went for renaming `wide` to `16x9` and `wide10` to `16x10` so it's very explicit. Having a query param with a value would need more work.